src/condor_utils/condor_holdcodes.h

```diff
@@ -206,7 +206,17 @@ BETTER_ENUM(CONDOR_HOLD_CODE, int,
 	JobRemoved = 1023,
 	ScratchDirError = 1024,
 	SuccessfulCheckpoint = 1025
-	// NOTE!!! If you add a new hold code here, don't forget to add a commas after all entries but the last!
+	,ActivationRefusedBadRequest = 1026      // activation request ad (job) has missing or invalid attributes
+	,ActivationRefusedNoMatch = 1027         // slot does not match (see Analyze attribute of reply)
+	,ActivationRefusedStillCleaning = 1028   // slot is still cleaning up from previous activation
+	,ActivationRefusedWorklifeExpired = 1029 // claim worklife expired
+	,ActivationRefusedPreempted = 1030       // claim was preempted
+	,ActivationRefusedBroken  = 1031         // slot is broken
+	,ActivationRefusedNotIdle = 1032         // slot is not idle and there is no better reason code.
+	,ActivationRefusedUnclaimed = 1033       // slot is not claimed
+	,ActivationRefusedClaimNotFound = 1034   // claim id was not found
```

Comment:
We should document these codes in `codes-other-values/vacate-reason-codes`

Reply:
done